### ISSUE TITLE
Buttons not clear in Suggestions

### ISSUE THREAD

**Issue by Archidoit** [LuatixHQ/opencti#9291]
## Description
The buttons and color are not clear in the Suggestion tab

## Reproducible Steps
- In a container, go in the Knowledge tab
- Open the suggestions
![Image](https://github.com/user-attachments/assets/0be14af3-1698-47f9-8f55-30252d2ed20e)

- Fill the form and eventually apply the suggestions
- If the suggestions has been applied the button is green, if not it is also green (but a different green). --> it's not very clear
![Image](https://github.com/user-attachments/assets/22d31970-a4ae-4ffd-95ae-d5e99f82e280)

## Expected behavior
Choose two different color for the button
Add a tooltip in the button describing the behavior

**Comment by nino-filigran** [comment]
I agree, we should opt for the green in suggestion (the lighter one)
